<launch>
    <arg name="rule_list"
         default="['$(find ghost-bridge)/rules/demo.ghost']" />
    <arg name="module_list"
         default="[$(find ghost-bridge)/scm/module1.scm]" />
    <rosparam param="ghost/rule_files" subst_value="True">$(arg rule_list) </rosparam>
    <rosparam param="ghost/scheme_modules" subst_value="True">$(arg module_list) </rosparam>
    <param name="ghost/response_wait_time" type="int" value="5" />
    <param name="ghost/relex_hostname" type="str" value="localhost" />
    <param name="ghost/relex_port" type="str" value="4444" />

    <node name="ghost_bridge" pkg="ghost-bridge" type="ghost-bridge_node" output="screen" />
</launch>
